docs/deployment/forked_threaded_servers.rst

```diff
@@ -0,0 +1,94 @@
+Forked and Threaded Servers
++++++++++++++++++++++++++++
+
+Forked and Threaded Servers share common "gotchas" and solutions when
+using pyramid and some popular packages.
+
+Forked and Threaded servers tend to use a "copy on write" implementation detail
+to optimize how they work and share memory. This can create problems when
+certain actions happen before the fork or thread dispatch, such as when files or
+file-descriptors are opened or random number generators are initialized.
+
+Many servers have built-in hooks or events which allow you to easily handle these
+situations.
+
+
+Servers
+++++++++++
```

Comment:
Duplicate primary heading level violates rST syntax. The primary heading is the chapter title (the file itself), and secondary and lower headings are sections, sub-sections, etc. Use another underlining character.

Also underlines should match the length of the text that they underline.